docs/_data/paths/temporary-aar-exceptions_{id}.yaml

```diff
@@ -81,7 +80,8 @@ put:
           $ref: ../../example-narratives/examples/temporary-aar-exception-get.json
   responses:
     '204': *no-content
-    '304': *not-modified
+    '304': &not-modified
```

Comment:
Actually, I just removed it from delete because it was clearly nonsense there as we discovered during implementation.  I only left it on PUT because I assumed it was there for a reason I didn't understand, but to me it seemed strange.  If it doesn't make sense to you either, I'm just going to remove it.

Reply:
Thanks, those 304s are my fault--I just checked. Copy/paste error during some yaml refactoring. I'm a little disappointed the linter didn't warn on this.